AVSynthesiser for Mac Engine (#350)
nateshmbhat/pyttsx3: docs/engine.rst

>       * `sapi5` - SAPI5 on Windows
>       * `nsss` - NSSpeechSynthesizer on Mac OS X
>       * `avsynth` - AVSynthesizer on Mac OS X (NSSS Is deprecated)
>       * `espeak` - eSpeak on every other platform

Reviewer: ```suggestion
      * `avsynth` - AVSynthesizer on macOS
      * `espeak` - eSpeak on all platforms
      * `nsss` - NSSpeechSynthesizer on macOS (Deprecated by Apple)
      * `sapi5` - SAPI5 on Windows
```
